js/bitopro.js

```diff
@@ -0,0 +1,1151 @@
+    }
+
+    parseOrder (order, market = undefined) {
+        //
+        // createOrder
+        //         {
+        //             orderId: '2220595581',
+        //             timestamp: '1644896744886',
+        //             action: 'SELL',
+        //             amount: '0.01',
+        //             price: '15000',
+        //             timeInForce: 'GTC'
+        //         }
+        //
+        // fetchOrder
+        //         {
+        //             "id":"8777138788",
+        //             "pair":"bnb_twd",
+        //             "price":"16000",
+        //             "avgExecutionPrice":"0",
+        //             "action":"SELL",
+        //             "type":"LIMIT",
+        //             "timestamp":1644899002598,
+        //             "status":4,
+        //             "originalAmount":"0.01",
+        //             "remainingAmount":"0.01",
+        //             "executedAmount":"0",
+        //             "fee":"0",
+        //             "feeSymbol":"twd",
+        //             "bitoFee":"0",
+        //             "total":"0",
+        //             "seq":"BNBTWD548774666",
+        //             "timeInForce":"GTC",
+        //             "createdTimestamp":1644898944074,
+        //             "updatedTimestamp":1644899002598
+        //         }
+        //
+        let id = this.safeString (order, 'orderId');
+        if (id === undefined) {
+            id = this.safeString (order, 'id');
+        }
+        const timestamp = this.safeInteger (order, 'timestamp');
+        const side = this.safeString (order, 'action');
+        const amount = this.safeString (order, 'amount');
+        const price = this.safeString (order, 'price');
+        const marketId = this.safeString (order, 'pair');
+        if (marketId !== undefined) {
+            market = this.market (marketId);
+        }
+        const symbol = this.safeString (market, 'symbol');
+        const orderStatus = this.safeString (order, 'status');
+        const status = this.parseOrderStatus (orderStatus);
+        const type = this.safeString (order, 'type');
+        const average = this.safeString (order, 'avgExecutionPrice');
+        const filled = this.safeString (order, 'executedAmount');
+        const remaining = this.safeString (order, 'remainingAmount');
+        const timeInForce = this.safeString (order, 'timeInForce');
+        let fee = undefined;
+        const feeAmount = this.safeString (order, 'fee');
+        const feeSymbol = this.safeString (order, 'feeSymbol');
+        if (feeAmount !== undefined) {
+            fee = {
+                'currency': feeSymbol,
+                'cost': feeAmount,
+            };
+        }
+        return this.safeOrder ({
+            'id': id,
+            'clientOrderId': undefined,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': undefined,
```

Comment:
Can't we fill this in with the `updatedTimestamp`?

Reply:
Yes